content/docs/use-cases/index.md

```diff
@@ -1,5 +1,14 @@
 # Use Cases
 
+# Collaboration Issues in Data Science
+
+Even with all the success we've seen today in machine learning (ML),
+specifically deep learning and its applications in business, the data science
+community still lacks good practices for organizing their projects and
+effectively collaborating across their varied ML projects. This is a critical
+challenge: we need to evolve towards ML algorithms and methods no longer being
+tribal knowledge and making them easy to implement, reuse, and manage.
+
```

Comment:
I see this was copied verbatim from content/docs/understanding-dvc/collaboration-issues.md. It may be a good first step but this task (#425) needs a longer process of understanding the content we're absorbing, and adapting it to its new context. Some of this wording may be repetitive for example, not sure. But I'm almost positive it won't work to just copy-paste it. Please try reading it in its new context.